docs/Readme.md

```diff
@@ -1,12 +1,15 @@
 # SynthDet Documentation
+<img src="images/Synthetic Data pipeline-Perception Workflow.png" align="middle"/>
```

Comment:
What is the difference between Dataset Evaluation and Model Evaluation? Afaict, there is only one type of evaluation, which evaluates a trained model on a target dataset. Also, Dataset Inspection/Statistics feeding a dataset into Dataset Evaluation is a bit odd. Consider drawing a box around Instpection/Statistics and Evaluation and label it datasetinsights. The dataset (that is, the directory of files) can be fed into any of those boxes.

Reply:
I guess it isn't clear if this is a data flow diagram or a user flow diagram.

Reply:
Updated diagrams per discussion